site/book/01-getting-started/01-system-requirements.md

```diff
@@ -13,9 +13,24 @@ $ kpt version
 
 kpt requires that you have [Git] installed and configured.
 
-## Docker
+## Container Runtime
 
-kpt requires that you have [Docker] installed and configured.
+`kpt` requires you to have at least one of the following runtimes installed and configured.
+
+Environment variable can be used to control which container runtime to use. More
+details can be found in the reference documents for [`kpt fn render`](/reference/cli/fn/render/)
+and [`kpt fn eval`](/reference/cli/fn/eval/).
+
+### Docker
+
+Here is the [instructions][install-docker] to have Docker installed and configured.
```

Comment:
Suggestion:

Please follow the [instructions] to install and configure Docker.

Reply:
Done